### ISSUE TITLE
Always wants to autocomplete \ref

### ISSUE THREAD

**Issue by GhostGambler** [Evpok/latex-autocomplete#12]
I have a line containing "section~\ref{something}" and whatever I type in the line behind that always triggers the dropdown to select a reference although the cursor is not inside a ref command. Even when I select something (up/down + enter) nothing happens and when I type the next character the dropdown pops up again.

One cannot deactivate the behavior in the options too.

latex-autocomplete version 0.5.2
Atom version 1.8.0

**Comment by Evpok** [comment]
Can you post a MWE ?

**Comment by GhostGambler** [comment]
```
Lorem Ipsum Section~\ref{sec:Statement of the Problem} Lorem Ipsum Section~\ref{sec:Review of Literature} Lorem Ipsum Section~\ref{sec:Aims / Research Questions} Lorem Ipsum Section~\ref{sec:Methodology} Lorem Ipsum Section~\ref{sec:Plan of Work}
```

**Comment by Evpok** [comment]
I couldn't reproduce that, but it put some other bugs in evidence. See if 0.5.3 fixes it for you.

**Comment by GhostGambler** [comment]
Sorry, I was stupid, this was a MWE for another bug for another plugin I filed in parallel…

Nevertheless the problem seems fixed in 0.5.3!
